queue/worker.go

```diff
@@ -140,6 +145,12 @@ func (r *Worker) call(task queue.Task) error {
 
 		r.printFailedLog(task, duration)
 
+		if tries < r.tries || r.tries == 0 { // If tries is 0, it means unlimited retries
```

Comment:
Shouldn't `r.failedJobChan <- models.FailedJob` only be executed after all retry attempts are exhausted and the job still fails?